warehouse/integrations/bigquery/bigquery.go

```diff
@@ -1216,15 +1256,19 @@ func (bq *BigQuery) TestLoadTable(ctx context.Context, location, tableName strin
 	gcsRef.MaxBadRecords = 0
 	gcsRef.IgnoreUnknownValues = false
 
-	partitionDate, err := bq.partitionDate()
+	partitioningInWarehouse, err := bq.fetchTablePartitionFromWarehouse(ctx, tableName)
```

Comment:
Don't think we need it for Validations. If the tables doesn't exist `fetchTablePartitionFromWarehouse` would error our right? Also, we delete the table post validations so would not be required.

Reply:
Idea is to keep the validations similar to the final load table. Why would the table doesn't exist? validation creates the table and then loads data?

Reply:
Post validation.